contrib/chinese/packages.el

```diff
@@ -14,13 +14,17 @@
 ;; which require an initialization must be listed explicitly in the list.
 (setq chinese-packages
     '(
-      youdao-dictionary
-      chinese-pyim
       find-by-pinyin-dired
       ace-pinyin
-      chinese-wbim
       ))
 
+(if chinese-enable-youdao-dict
+  (add-to-list 'chinese-packages 'youdao-dictionary))
```

Comment:
In other layers, we have used `push` instead of `add-to-list` for this same purpose. If I remember correctly, the Emacs documentation also recommends using `push` most of the time.